src/server/rest/v1/service/ChargingStationService.ts

```diff
@@ -1108,12 +1108,43 @@ export default class ChargingStationService {
         message: 'Charging Station is not connected to the backend',
       });
     }
-
     const result = await chargingStationClient.reserveNow(filteredRequest.args);
     res.json(result);
     next();
   }
 
+  public static async handleCancelReservation(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+    // Check auth
+    if (!await Authorizations.canListChargingStations(req.user)) {
```

Comment:
No need this auth, it's already checked in UtilsService.checkAndGetChargingStationAuthorization()
To remove everywhere.